### ISSUE TITLE
fontconfig seems to require graphite as a dependency

### ISSUE THREAD

**Issue by saltedcoffii** [skycocker/chromebrew#5158]
```
fontconfig: Fontconfig is a library for configuring and customizing font access.
Precompiled binary available, downloading...
############################################################################ 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
Performing pre-install...
Performing install...
Performing post-install...
Putting fontconfig code in ~/.bashrc
To complete the installation, execute the following:
source ~/.bashrc
fc-cache: error while loading shared libraries: libgraphite2.so.3: cannot open shared object file: No such file or directory
```

I'm not sure what you guys want to do about this, whether add `graphite` as a dependency or change fontconfig's postinstall or something else about the binary.

**Comment by booperlv** [comment]
this prevented a whole lot of programs from installing for me, is there any working solution right now?

**Comment by uberhacker** [comment]
What happens if you `crew install graphite && crew reinstall fontconfig -s`?  If that works, then we can add the dependency and bump the version.

**Comment by saltedcoffii** [comment]
@uberhacker it works.

**Comment by uberhacker** [comment]
Great, did you want to go ahead and submit a PR?

**Comment by saltedcoffii** [comment]
This issue doesn't seem to be fixed, #5160 perpetuates it.
